From a0a45fa9810f8b72066bcdd2540e81a6e3654ae0 Mon Sep 17 00:00:00 2001
From: Walter Lee <waltl@google.com>
Date: Fri, 5 Jan 2024 22:08:40 -0500
Subject: [PATCH] Define constraint for emscripten os

This is to be used for Web Assembly, with cpus wasm32/wasm64.
---
 os/BUILD | 5 +++++
 1 file changed, 5 insertions(+)

diff --git a/os/BUILD b/os/BUILD
index ae6e8b0..8e9f990 100644
--- a/os/BUILD
+++ b/os/BUILD
@@ -106,6 +106,11 @@ constraint_value(
     constraint_setting = ":os",
 )

+constraint_value(
+    name = "emscripten",
+    constraint_setting = ":os",
+)
+
 # WASI (WebAssembly System Interface)
 # https://github.com/bytecodealliance/wasmtime/blob/main/docs/WASI-overview.md
 constraint_value(
